tests/utils/olp_server/server.js

```diff
@@ -29,11 +29,80 @@ const blob_service_handler = require('./blob_service.js')
 
 const port = 3000
 
-const sleep = (milliseconds) => {
-    return new Promise(resolve => setTimeout(resolve, milliseconds))
+const sleep =
+    (milliseconds) => {
+      return new Promise(resolve => setTimeout(resolve, milliseconds))
+    }
+
+
+
+function handleRequest(response, pathname, request, handler) {
+  const result = handler(pathname, request)
+  response.writeHead(result.status, result.headers)
+  response.end(result.text)
+}
+
+function
+getError() {
```

Comment:
```suggestion
generateErrorAtRandom() {
```
Or `getErrorAtRandom`. IMHO, `getError` is too basic name